From a1205f7ae980146f68da6ac935024247bd2cd89a Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Sat, 8 Apr 2017 15:09:00 +0200
Subject: [PATCH 56/74] libfm_jni: Add jni core headers dependency

 * Due to a few moved core jni headers, add the new include path

Change-Id: I9a1c5885d1c14151d2f3303e429081d1fc214a7d
---
 libfm_jni/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libfm_jni/Android.mk b/libfm_jni/Android.mk
index 468006e..5ac0135 100644
--- a/libfm_jni/Android.mk
+++ b/libfm_jni/Android.mk
@@ -13,6 +13,7 @@ LOCAL_SRC_FILES := \
     LibfmJni.cpp
 
 LOCAL_C_INCLUDES := $(JNI_H_INCLUDE) \
+    frameworks/base/core/jni/include \
     frameworks/base/include/media
 
 LOCAL_SHARED_LIBRARIES := \
-- 
2.17.1

